tests/workbenches/notebook-controller/test_spawning.py

```diff
@@ -0,0 +1,201 @@
+        Step(
+            value="Delete ODH operator and all created resources",
+            expected="Operator is removed and all other resources as well",
+        )
+    },
+)
+class TestNotebookST:
+    @classmethod
+    @property
+    @functools.cache
+    def logger(cls):
+        return logging.getLogger(cls.__name__)
+
+    DS_PROJECT_NAME: str = "test-notebooks"
+
+    NTB_NAME: str = "test-odh-notebook"
+    NTB_NAMESPACE: str = "test-odh-notebook"
+
+    @TestDoc(
+        description=Desc("Create simple Notebook with all needed resources and see if Operator creates it properly"),
+        contact=Contact(name="Jakub Stejskal", email="jstejska@redhat.com"),
+        steps={
+            Step(
+                value="Create namespace for Notebook resources with proper name, labels and annotations",
+                expected="Namespace is created",
+            ),
+            Step(value="Create PVC with proper labels and data for Notebook", expected="PVC is created"),
+            Step(
+                value="Create Notebook resource with Jupyter Minimal image in pre-defined namespace",
+                expected="Notebook resource is created",
+            ),
+            Step(
+                value="Wait for Notebook pods readiness",
+                expected="Notebook pods are up and running, Notebook is in ready state",
+            ),
+        },
+    )
+    def testCreateSimpleNotebook(self, function_resource_manager, admin_client, unprivileged_client):
+        with allure.step("Create namespace"):
+            ns: ocp_resources.namespace.Namespace = ocp_resources.namespace.Namespace(
+                name=self.NTB_NAMESPACE,
+                label={OdhAnnotationsLabels.LABEL_DASHBOARD: "true"},
+                annotations={OdhAnnotationsLabels.ANNO_SERVICE_MESH: "false"},
+            )
+            function_resource_manager.createResourceWithoutWait(unprivileged_client, ns)
+
+        with allure.step("Create PersistentVolumeClaim"):
+            pvc: ocp_resources.persistent_volume_claim.PersistentVolumeClaim = (
+                ocp_resources.persistent_volume_claim.PersistentVolumeClaim(
+                    name=self.NTB_NAME,
+                    namespace=self.NTB_NAMESPACE,
+                    label={OdhAnnotationsLabels.LABEL_DASHBOARD: "true"},
+                    accessmodes="ReadWriteOnce",
+                    size="10Gi",
+                    volume_mode=ocp_resources.persistent_volume_claim.PersistentVolumeClaim.VolumeMode.FILE,
+                )
+            )
+            function_resource_manager.createResourceWithoutWait(unprivileged_client, pvc)
+
+        with allure.step("Create Notebook CR"):
+            # notebookImage: str = NotebookType.getNotebookImage(NotebookType.JUPYTER_MINIMAL_IMAGE, NotebookType.JUPYTER_MINIMAL_2023_2_TAG);
+            notebookImage: str = "quay.io/modh/odh-minimal-notebook-container@sha256:615af25cfd4f3f2981b173e1a5ab24cb79f268ee72dabbddb6867ee1082eb902"
```

Comment:
Can this be fetched dinamically from the cluster?

Reply:
yes, I changed the approach, thanks